Makefile

```diff
@@ -397,6 +397,7 @@ $(shell mkdir -p $(foreach dir, \
 endif
 
 ifeq ($(COMPILER),ido)
+$(BUILD_DIR)/src/boot/logutils.o: OPTFLAGS := -O2
```

Comment:
This change doesn't affect the other versions, right? I don't remember if we are checking them all in jenkins yet

Reply:
we are checking all versions with jenkins (thanks frug :frog: )

other versions besides oot-gc-mq-dbg are non-debug, so the functions are so even more dead simple -g3 probably doesn't even matter anyway